Makefile

```diff
@@ -7,59 +7,71 @@ SHELL := bash
 .SUFFIXES:
 .SECONDARY:
 
-OBO = http://purl.obolibrary.org/obo/
-
-CORE = obo-core.owl
-MODULES = generic physical biological human science
+ROBOT := java -jar build/robot.jar
 
-PREFIX = 'CORE: http://purl.obolibrary.org/CORE_'
+build/robot.jar:
+	curl -Lk -o $@ https://github.com/ontodev/robot/releases/download/v1.4.3/robot.jar
 
-GENERIC_SHEET = https://docs.google.com/spreadsheets/d/e/2PACX-1vRGrxd10VuAmb55RqWEzft8q64mI0Ryr8biOb3K8Sx281Xv0NyRVwhr-Z_0IjFWra8dPmHYeKng6PbS/pub?gid=1709706289&single=true&output=tsv
-PHYSICAL_SHEET = https://docs.google.com/spreadsheets/d/e/2PACX-1vRGrxd10VuAmb55RqWEzft8q64mI0Ryr8biOb3K8Sx281Xv0NyRVwhr-Z_0IjFWra8dPmHYeKng6PbS/pub?gid=2144007357&single=true&output=tsv
-BIOLOGICAL_SHEET = https://docs.google.com/spreadsheets/d/e/2PACX-1vRGrxd10VuAmb55RqWEzft8q64mI0Ryr8biOb3K8Sx281Xv0NyRVwhr-Z_0IjFWra8dPmHYeKng6PbS/pub?gid=663651851&single=true&output=tsv
-HUMAN_SHEET = https://docs.google.com/spreadsheets/d/e/2PACX-1vRGrxd10VuAmb55RqWEzft8q64mI0Ryr8biOb3K8Sx281Xv0NyRVwhr-Z_0IjFWra8dPmHYeKng6PbS/pub?gid=1499631141&single=true&output=tsv
-SCIENCE_SHEET = https://docs.google.com/spreadsheets/d/e/2PACX-1vRGrxd10VuAmb55RqWEzft8q64mI0Ryr8biOb3K8Sx281Xv0NyRVwhr-Z_0IjFWra8dPmHYeKng6PbS/pub?gid=1951374411&single=true&output=tsv
+OBO = http://purl.obolibrary.org/obo/
+COB = cob.owl
+MODULES = generic physical biological human science
+PREFIX = 'COB: http://purl.obolibrary.org/COB_'
```

Comment:
Should be `http://purl.obolibrary.org/obo/COB_` -- missing the `/obo/`. Please check everywhere for this mistake.